front/src/reducers/osrdsimulation/types.ts

```diff
@@ -149,16 +176,22 @@ export interface SimulationTrain<DateType = Date> {
 
 export interface OsrdSimulationState {
   redirectToGraph?: boolean;
-  chart: any;
-  chartXGEV: any;
-  contextMenu: any;
-  hoverPosition: any;
+  chart?: Chart;
+  chartXGEV?: Chart['x'];
```

Comment:
Do you know why we store the chart in the store ?

Reply:
I think it’s the current zoom/current x y position on the map, this sort of things.